.github/workflows/mattermost-ziti-webhook.yml

```diff
@@ -1,24 +1,31 @@
 name: mattermost-ziti-webhook
-on: 
-  create:
-  delete:
+on:
   issues:
   issue_comment:
   pull_request_review:
+    types: [submitted]
```

Comment:
The `if` check (line 24) will prevent any `pull_request_review` other than `approved` from posting to Mattermost

Reply:
nvm - I got that wrong... (the if check is for state, not action...)